Notebooks/Metrics.ipynb

```diff
@@ -0,0 +1,531 @@
+{
```

Comment:
<p>If you want to be consistent, should do</p><p>"get_distance_air(lat_a, long_a, lat_b, long_b) * 1.3"</p><p>instead of</p><p>"return geodesic((lat_a, long_a), (lat_b, long_b)).km * 1.3"</p><p>in line 51</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AndOleAnd/Capstone_N_A_P/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>